frontend/src/views/DriveFormView.vue

```diff
@@ -329,6 +337,7 @@ export default {
     ...mapState(namespaces.passengers, {
       passengers: state =>
         (state.data || []).map(p => ({
+          ...p,
```

Comment:
```suggestion
          rsaPubE,
          rsaModulusN
```

Reply:
I don't quite get how to do it like you suggested. I can't just list `rsaPubE`, `rsaModulusN` like that can I?
I did the usual `rsaPubE: p.rsaPubE`, hope that's what you meant.

Reply:
Right, my bad, maybe something like this:

```js
(state.data || []).map(
  ({id, firstName, lastName, rsaModulusN, rsaPubE}) => ({
          value: id,
          text: [firstName, lastName].join(' '),
          rsaModulusN,
          rsaPubE,
  })),
```

Reply:
whoa, that's wild! :D :dancer: Will test it later and see how it goes

Reply:
After second thought - don't you think that the current version is good enough?

Reply:
it's ok